.gitignore

```diff
@@ -2,12 +2,15 @@
 .xhtml
 .xml
 .byebug_history
+.DS_Store
```

Comment:
A [global `~/.gitignore` file](https://davidwalsh.name/global-gitignore) can also help prevent these OS-specific files from accidentally getting added to repos (like `Thumbs.db` for Windows users).